packages/view/src/ui-view.tsx

```diff
@@ -1,15 +1,46 @@
 import React from 'react';
 
-import styled from 'styled-components';
+import styled, { createGlobalStyle } from 'styled-components';
 
-import { ThemeContext, getThemeStyling, UiViewport, Breakpoints } from '@uireact/foundation';
+import {
+  ThemeContext,
+  getThemeStyling,
+  UiViewport,
+  Breakpoints,
+  getTextSize,
+  TextSize,
+  getThemeColor,
+  ColorCategories,
+  ColorTokens,
+} from '@uireact/foundation';
 
-import { UiViewProps } from './types/ui-view-props';
+import { UiViewProps, privateViewProps } from './types/ui-view-props';
 import { themeMapper } from './theme';
 import { CenteredDiv } from './__private';
 
-const Div = styled.div<UiViewProps>`
-  ${(props) => getThemeStyling(props.theme, props.selectedTheme, themeMapper)}
+const GlobalStyle = createGlobalStyle<privateViewProps>`
+  * {
+    margin: 0;
+    padding: 0;
+  }
+
+  body {
+    ${(props) => `
+      ${`font-size: ${getTextSize(props.customTheme, TextSize.regular)};`}
+      ${`font-family: ${props.customTheme.texts.font};`}
+      ${`background-color: ${getThemeColor(
+        props.customTheme,
+        props.selectedTheme,
+        ColorCategories.backgrounds,
+        ColorTokens.token_100
+      )};`}
+    `}
+    font-weight: 400;
+  }
```

Comment:
I was also missing the global styles which gives the styling for fonts, sizes and colors at a page level which is what the list item uses for displaying the correct color.